tsconfig-base.json

```diff
@@ -3,8 +3,13 @@
 {
   "compilerOptions": {
     "composite": true,
+
+    // Set up incremental builds. All sub-projects emit under outDir, maintaining
+    // directory structure relative to rootDir. No effect on include, exclude, etc.
     "emitDeclarationOnly": true,
     "declarationMap": true,
+    "outDir": "./.tmp/tsbuildinfo/",
```

Comment:
nit: i'm recently of the opinion that leading `./` in paths are pointless noise. if a path is not starting with `/` I know it is relative :)

Reply:
I don't really care, it just bothers me that `rootDir: ''` isn't the identity instead of `'.'` in this case, since they're right next to each other :)